cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -757,6 +757,21 @@ function Configure-HostNetworkingService {
   Log-Output "Host network setup complete"
 }
 
+function Configure-GcePdTools {
+  if (ShouldWrite-File ${env:K8S_DIR}\GetGcePdName.dll) {
+    MustDownload-File -OutFile ${env:K8S_DIR}\GetGcePdName.dll `
+      -URLs "https://github.com/pjh/gce-tools/raw/master/GceTools/GetGcePdName/GetGcePdName.dll"
+  }
+  if (-not (Test-Path $PsHome\profile.ps1)) {
+    New-Item -path $PsHome\profile.ps1 -type file
+  }
+
+    Add-Content $PsHome\profile.ps1 `
```

Comment:
Why do we need to set this in the profile?

Reply:
Import-Module will only work for the current powershell session. When later Get-GcePdName is executed in a different session, it will not work. Setting to this profile will make all users and all sessions can access the command